app/models/forum_post.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class ForumPost < ApplicationRecord
```

Comment:
Just override the relevant methods, no need to override entire methods, refer https://guides.rubyonrails.org/engines.html#overriding-models-and-controllers